jobs/update-payments.js

```diff
@@ -1,23 +1,16 @@
-const dbs = require('../lib/dbs')
 const env = require('../lib/env')
-const { getActiveBilling } = require('../lib/payments')
+const { getActiveBilling, getCurrentlyPrivateAndEnabledRepos } = require('../lib/payments')
 const stripe = require('stripe')(env.STRIPE_SECRET_KEY)
-const _ = require('lodash')
 
-module.exports = async ({ accountId }) => {
-  const { repositories } = await dbs()
+module.exports = async ({ accountId, repositoryId }) => {
   const billingAccount = await getActiveBilling(accountId)
-  if (!billingAccount || !billingAccount.stripeItemId) return
+    // ignore non-stripe users
+  if (!billingAccount || !billingAccount.stripeSubscriptionId) return
 
-  const billing = await repositories.query('billing', {
-    key: accountId,
-    group_level: 1,
-    reduce: true
-  })
-  const currentlyPrivateAndEnabledRepos = _.get(billing, 'rows[0].value', 0)
+  const currentlyPrivateAndEnabledRepos = getCurrentlyPrivateAndEnabledRepos(accountId)
```

Comment:
The test for this job fails because `getCurrentlyPrivateAndEnabledRepos is not a function` .. I can't see why

Reply:
it’s because of `.noCallThru()`: https://www.npmjs.com/package/proxyquire#prevent-call-thru-for-all-future-stubs-resolved-by-a-proxyquire-instance

Reply:
remove the `.noCallThrough()` from `require('proxyquire')`. You’ll also need to change this line to 

```js
const currentlyPrivateAndEnabledRepos = await getCurrentlyPrivateAndEnabledRepos(accountId)
```